src/resolvers/index.ts

```diff
@@ -196,6 +225,34 @@ const resolvers = {
 
   // Root resolvers
   Query: {
+    guardianDirectory: async (
+      _: undefined,
+      args: undefined,
+      { dataSources }: PortalUserContext
+    ) => {
+      const {
+        data: { getGuardianDirectory },
+      } = await dataSources.personnelAPI.getGuardianDirectory()
+
+      const guardianDirectory: GuardianDirectory[] = []
+      // Pull out and format relevant data to return for directory
```

Comment:
@gidjin I noticed the formatting is gone and all the text is rendering as-is from the spreadsheet / mostly in caps. This might not be the best place to handle it (where I had it before), but maybe we should add it back somewhere else? Plain CSS wasn't working for me. Lmk what you think!

Reply:
never mind! i just saw it's in the personnel-api branch -- missed it before :) thank you!

Reply:
The formatting moved to the personnel API see PR https://github.com/USSF-ORBIT/ussf-personnel-api/pull/18. I did that because I don't know what data formatting we will need to do once we have access to the real API we may still have to do data transformation and it felt best to do that in our personnel API since I see it as place to transform external APIs into what we need for this. Sorry for not flagging that in the description! If you have questions or can't get that to work let me know